dpti/workflows/simulations/ti_sim.py

```
@@ -0,0 +1,683 @@
+    # NODEDATA_FILENAME = "ti_settings.json"
+    # node_settings: TISimulationSettings
+    nodedata_type: Type[TISimulationNodeData] = TISimulationNodeData
+    workflow_services: BasicWorkflowServices
+    # def __init__(self, updates={}, template_json:Optional[str]=None):
+    #     self.updates = updates
+    #     self.template_json = template_json
+    #     self.updated_nodedata = self.nodedata_type.from_template(
+    #         updates=updates, template_json=template_json)
+
+    # @task
+    def _prepare(self) -> Dict[str, Any]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)